apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx

```diff
@@ -0,0 +1,121 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import * as Table from 'reactabular-table';
+import {lessonEditorTableStyles} from './TableConstants';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  actionsColumn: {
+    display: 'flex',
+    justifyContent: 'space-evenly',
+    backgroundColor: 'white'
+  },
+  remove: {
+    fontSize: 14,
+    color: 'white',
+    background: color.dark_red,
+    cursor: 'pointer',
+    textAlign: 'center',
+    width: '50%',
```

Comment:
Are you leaving this at 50% because you will be adding the add button in the next PR?

Reply:
oops - copy paste error. there were two buttons in the place I copied this from. I will fix. no current plans to add an "add" button soon. we'll use the dropdowns for that.

Reply:
I changed the styles to get rid of some of the whitespace:
![Screen Shot 2021-03-25 at 11 02 55 AM](https://user-images.githubusercontent.com/8001765/112521529-d9c36880-8d59-11eb-8c41-9caf556ea9dc.png)